src/js/Edit/L.PM.Edit.Circle.js

```diff
@@ -10,6 +10,9 @@ Edit.Circle = Edit.extend({
     this._updateHiddenPolyCircle();
   },
   enable(options) {
+    // backwards compatibility
+    options.allowEditing = options.radiusEdit ?? true;
```

Comment:
`allowEditing` enables the editing in generally, so this line should be remove. Editing with `radiusEdit: false` should be the same as dragging but in editMode

Reply:
I must have been staring at this PR for too long... should be (and has been fixed in latest commit) `options.radiusEdit = options.radiusEdit ?? true`

Reply:
If there's somewhere better to set this like there was with L.PM.Draw, let me know!